src/common/entity/domain_icon.ts

```diff
@@ -88,6 +87,9 @@ export const domainIcon = (domain: string, state?: string): string => {
           return "hass:window-open";
       }
 
+    case "device_tracker":
+      return state && state === "home" ? "hass:lan-connect" : "hass:lan-disconnect";
```

Comment:
Should we check the state attribute `"source_type"` too and only use these icons if that attribute is `"router"`. There are different categories of device trackers. Eg GPS trackers should perhaps not be represented with the lan icons?

Reply:
ok, thats a good idea, thx for the suggestion

Reply:
I'm not a frontend expert. Someone in the frontend team should continue the review. 👍

Reply:
Your branch is old, this file does not look anything like this anymore.
If you rebase (or start over) with the current dev, how to do this based on attributes should be clearer. (Look at cover for instance)